src/postgraphql/__tests__/pathgraphqlIntegrationSchemaExport-test.js

```diff
@@ -0,0 +1,35 @@
+jest.mock('fs')
```

Comment:
Did you mean to call this file `pathgraphqlIntegrationSchemaExport` instead of `postgraphqlIntegrationSchemaExport`?